### PR TITLE
Magn 8960   freeze ui changes

### PR DESCRIPTION
**Pull request by ramramps**
## Purpose

The purpose of this PR is to change the appearance of the frozen nodes and its connectors.
capture

This PR refers to the below tasks 
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-8959

## Design Document
 https://docs.google.com/a/adsk-oss.com/presentation/d/1MUhSzoDo-QFPysDv1E6gRslpq2pnb6eoPjKSsJN6g-o/edit?usp=sharing  

## Declaration
Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files

## Reviewers
@mjkkirschner

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs [hunk 1]
```diff
@@ -219,6 +219,11 @@ public PreviewState PreviewState
                 return PreviewState.None;
             }
         }
+         
+        public bool IsFrozen
```
Comment (ramramps) [inline src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs]: Connector gets the state of the node.

File: src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs [hunk 2]
```diff
@@ -1062,6 +1064,22 @@ private bool CanToggleIsFrozen(object parameters)
             return DynamoSelection.Instance.Selection.Count() == 1;
         }
 
+        /// <summary>
+        /// When a node is frozen, raise the IsFrozen property changed event on
+        /// all its downstream nodes, to ensure UI updates correctly.
+        /// </summary>
+        private void RaisePropertyChangedOnDownStreamNodes()
```
Comment (ramramps) [inline src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs]: Frozen is not directly set on the downstream nodes. So, raise the property changed event on downstream nodes. 
This is raised only when the user explicitly freeze a node. So there will not be any performance change.

File: src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs [hunk 3]
```diff
@@ -1062,6 +1064,22 @@ private bool CanToggleIsFrozen(object parameters)
...
+        {
+            HashSet<NodeModel> nodes = new HashSet<NodeModel>();
+            this.nodeLogic.GetDownstreamNodes(this.nodeLogic, nodes);
+
+            foreach (var inode in nodes)
+            {
+                var current = this.WorkspaceViewModel.Nodes.FirstOrDefault(x => x.NodeLogic == inode);
+                if (current != null) current.RaisePropertyChanged("IsFrozen");
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs]: can we add brackets here, just a style thing.

**Comment by ramramps** [comment]
@mjkkirschner - Can I merge this now?

**Comment by mjkkirschner** [comment]
@ramramps yep, looks good to me :)

**Pull request merged**